app/page/info.tsx

```diff
@@ -140,133 +132,89 @@ const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
   }
 }
 
-enum transactionType {
-  normal = "normal",
-  internal = "internal"
-}
-
-const accountTransactions = async (
-  explorerUrl: string,
-  address: HexString
-): Promise<string[]> => {
-  return transactionsCrawler(
-    transactionType.normal,
-    `${explorerUrl}address/${address}`
-  )
+// getUserOpsUiData() filters UserOperationLog objects based on a specific
+// sender address and transforms them into UserOperationUi objects
+// containing status and hash properties.
+const getUserOpsUiData = (
+  userOpLogs: UserOperationLog[],
+  chainId: number,
+  accountAddress: string
+): UserOperationUi[] => {
+  return userOpLogs
+    .filter((userOpLog) => {
+      const userOp = new UserOperation(userOpLog.userOp)
+      return userOp.sender.toLowerCase() === accountAddress.toLowerCase()
+    })
+    .map((userOpLog) => {
+      const userOp = new UserOperation(userOpLog.userOp)
+      return {
+        status: userOpLog.status,
+        hash: userOp.hash(userOpLog.entryPointAddress, chainId)
```

Comment:
`UserOperationLog` has already kept the hash for you, don't need to recalculate.

Reply:
It is necessary to determine whether userOpLog.status is Sent, Succeeded, or Failed to retrieve userOpHash from userOpLog.receipt.
Otherwise, TypeScript will throw an error due to userOpLog.receipt being undefined if userOpLog.status is Pending or Rejected.
Therefore, the simplest solution is to recalculate the userOpHash value.